src/table/cells/handle-cell-focus.js

```diff
@@ -14,25 +14,29 @@ export const updateFocus = (rowElements, cellCoord, shouldFocus = true) => {
 export const handleClickToFocusBody = (cell, focusedCellCoord, rootElement) => {
   const { rawRowIdx, rawColIdx } = cell;
   const rowElements = rootElement.getElementsByClassName('sn-table-row');
-  updateFocus(rowElements, focusedCellCoord.current, false);
-  focusedCellCoord.current = [rawRowIdx + 1, rawColIdx];
-  updateFocus(rowElements, [rawRowIdx + 1, rawColIdx]);
+  updateFocus(rowElements, focusedCellCoord, false);
+  updateFocus(rowElements, [rawRowIdx + 1, rawColIdx], true);
```

Comment:
Shouldn't `focusedCellCoord` be updated here, i.e. call `setFocusedCellCoord`

Reply:
✅ Fixed

Reply:
also updated the test cases to check if `setFocusedCellCoord` has been called ( of course it does and passes the unit test ✅ )